### ISSUE TITLE
Status Bars: Undesribable behaviour after 10.2

### ISSUE THREAD

**Issue by ls-** [Stanzilla/WoWUIBugs#498]
In 10.2 the status bar behaviour was changed.

Right now when the current value is set to whatever the min value is, it breaks all the anchors of whatever is attached to the status bar texture, in this example it's a secondary status bar:

![bug_sb_pv](https://github.com/Stanzilla/WoWUIBugs/assets/2725970/c8513d5a-28d1-49cc-8ccf-e3bfb3d97ae5)

That said, if you set the bar value to any value that isn't min value afterwards, the secondary bar will reattach itself properly again.

Moreover, it seems that instead of being actually empty, the status bar's texture is simply invisible, but its width is set to the max instead of being ~0, so if you try to attach anything to it, it'll look like this:

![image](https://github.com/Stanzilla/WoWUIBugs/assets/2725970/430a0803-2e74-48d8-9213-15dec60fbde1)

For context, this is the pre-10.2 behaviour. I could reproduce it by setting the min value to -0.001 instead of 0, any value that's just a tiiiiny bit lower than the desired min value should do the trick here. So at least there's a workaround, I guess.

![old_sb_pv](https://github.com/Stanzilla/WoWUIBugs/assets/2725970/7e9c9c7f-b4e6-497a-93f2-9a3054910fa9)

**Comment by kodewdle** [comment]
```lua
/run X=PlayerFrame.PlayerFrameContent.PlayerFrameContentMain.HealthBarArea.HealthBar

/run Y=CreateFrame('StatusBar', '$parent_RedBar', X);Y:SetSize(100, 20);Y:SetPoint('LEFT', X:GetStatusBarTexture(), 'RIGHT');Y:SetStatusBarTexture([[Interface\Buttons\White8x8]]);Y:SetStatusBarColor(1, 0, 0, 1);Y:SetMinMaxValues(0, 1);Y:SetValue(0)

/run W=CreateFrame('StatusBar', '$parent_GreenBar', X);W:SetSize(100, 20);W:SetPoint('LEFT', Y:GetStatusBarTexture(), 'RIGHT');W:SetStatusBarTexture([[Interface\Buttons\White8x8]]);W:SetStatusBarColor(0, 1, 0, 1);W:SetMinMaxValues(0, 6);W:SetValue(3)
```

do those 3:
![image](https://github.com/Stanzilla/WoWUIBugs/assets/2266754/f3d8c899-1b0b-4ff7-bbea-e09f0fdedb9c)

then do: `/run Y:SetValue(0.1)`: 
![image](https://github.com/Stanzilla/WoWUIBugs/assets/2266754/689ba03a-4734-4e37-ae8f-b426baa41f3c)

then do: `/run Y:SetValue(0)`: 
![image](https://github.com/Stanzilla/WoWUIBugs/assets/2266754/a927e421-912a-4225-9fc4-0bbbdff75558)

then do: `/run Y:SetValue(0.0001)`
![image](https://github.com/Stanzilla/WoWUIBugs/assets/2266754/ae7e437d-8e19-43e3-a45a-18c8f7e143c3)

then do: `/run Y:SetValue(0)` and `/run Y:SetStatusBarTexture([[Interface\Buttons\White8x8]])`
![image](https://github.com/Stanzilla/WoWUIBugs/assets/2266754/ff43b4aa-9c8d-484c-9d62-f8df6a062ab5)

last one: when the `value` is the same as the `min` value and the texture is changed, the position of the `GetStatusBarTexture` is reset to full.

**Comment by kodewdle** [comment]
this also effects Classic Era 1.15

**Comment by ls-** [comment]
The bug is fixed in 10.2.5, and I hope it stays this way 🙏🏻

![bug_sb_fixed_pv](https://github.com/Stanzilla/WoWUIBugs/assets/2725970/06dba8c7-d50a-4734-bed2-a25e89655a19)

**Comment by Meorawr** [comment]
Marking as fixed based upon the 10.2.5 feedback.
